Tasks/AzurePowerShellV3/AzurePowerShell.ps1

```diff
@@ -74,10 +78,27 @@ catch
 
 Update-PSModulePathForHostedAgent -targetAzurePs $targetAzurePs -authScheme $authScheme
 
+# troubleshoot link
+$troubleshoot = "https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-powershell?view=azure-devops#troubleshooting"
+$status = $true
 try {
     # Initialize Azure.
     Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
     Initialize-Azure -azurePsVersion $targetAzurePs -strict
+    Write-Host "## Stage 2: Initialize Azure Complete"
+} 
+catch {
```

Comment:
don't need catch here. In 'finally' block, check for whether or not status is exists. That will work.